node/pkg/driver/node_test.go

```diff
@@ -1138,74 +1139,105 @@ func TestNodeGetInfo(t *testing.T) {
+		fcExists        bool
+		iscsiExists     bool
 	}{
 		{
-			name:          "good iqn, empty fc with error from node_utils",
-			return_fc_err: fmt.Errorf("some error"),
-			expErr:        status.Error(codes.Internal, fmt.Errorf("some error").Error()),
-			iscsiExists:   true,
-			fcExists:      true,
+			name:        "good iqn, empty fc with error from node_utils",
+			returnFcErr: fmt.Errorf("some error"),
+			expErr:      status.Error(codes.Internal, fmt.Errorf("some error").Error()),
+			nvmeExists:  false,
+			fcExists:    true,
+			iscsiExists: true,
+		},
+		{
+			name:         "empty nqn with error from node_utils, one more fc ports",
+			returnNqnErr: fmt.Errorf("some error"),
+			returnFcs:    []string{"10000000c9934d9f", "10000000c9934d9h"},
+			expNodeId:    "test-host;;10000000c9934d9f:10000000c9934d9h",
+			nvmeExists:   true,
+			fcExists:     true,
+			iscsiExists:  true,
 		},
 		{
 			name:        "empty iqn with error, one fc port",
-			return_fcs:  []string{"10000000c9934d9f"},
-			expNodeId:   "test-host;10000000c9934d9f",
-			iscsiExists: true,
+			returnFcs:   []string{"10000000c9934d9f"},
+			expNodeId:   "test-host;;;10000000c9934d9f",
+			nvmeExists:  false,
 			fcExists:    true,
+			iscsiExists: true,
 		},
 		{
 			name:        "empty iqn with error from node_utils, one more fc ports",
-			return_iqn:  "",
-			return_fcs:  []string{"10000000c9934d9f", "10000000c9934d9h"},
-			expNodeId:   "test-host;10000000c9934d9f:10000000c9934d9h",
-			iscsiExists: true,
+			returnIqn:   "",
+			returnFcs:   []string{"10000000c9934d9f", "10000000c9934d9h"},
+			expNodeId:   "test-host;;10000000c9934d9f:10000000c9934d9h",
+			nvmeExists:  false,
 			fcExists:    true,
+			iscsiExists: true,
 		},
 		{
 			name:        "good iqn and good fcs",
-			return_iqn:  "iqn.1994-07.com.redhat:e123456789",
-			return_fcs:  []string{"10000000c9934d9f", "10000000c9934d9h"},
-			expNodeId:   "test-host;10000000c9934d9f:10000000c9934d9h;iqn.1994-07.com.redhat:e123456789",
+			returnIqn:   "iqn.1994-07.com.redhat:e123456789",
+			returnFcs:   []string{"10000000c9934d9f", "10000000c9934d9h"},
+			expNodeId:   "test-host;;10000000c9934d9f:10000000c9934d9h;iqn.1994-07.com.redhat:e123456789",
+			nvmeExists:  false,
+			fcExists:    true,
 			iscsiExists: true,
+		},
+		{
+			name:        "good nqn, good fcs and good iqn",
+			returnNqn:   "nqn.2014-08.org.nvmexpress:uuid:b57708c7-5bb6-46a0-b2af-9d824bf539e1",
+			returnIqn:   "iqn.1994-07.com.redhat:e123456789",
+			returnFcs:   []string{"10000000c9934d9f", "10000000c9934d9h"},
+			expNodeId:   "test-host;nqn.2014-08.org.nvmexpress:uuid:b57708c7-5bb6-46a0-b2af-9d824bf539e1;10000000c9934d9f:10000000c9934d9h;iqn.1994-07.com.redhat:e123456789",
+			nvmeExists:  true,
 			fcExists:    true,
+			iscsiExists: true,
 		},
 		{
-			name:        "iqn and fc path are inexistent",
+			name:        "nqn, fc and iqn path are inexistent",
 			iscsiExists: false,
+			nvmeExists:  false,
 			fcExists:    false,
```

Comment:
```suggestion
			nvmeExists:  false,
			fcExists:    false,
			iscsiExists: false,
```

Reply:
done

Reply:
[nope](https://github.com/IBM/ibm-block-csi-driver/blob/b2f7c62317d3e3471a6d2341122800b02cde0100/node/pkg/driver/node_test.go#L1201-L1203)

Reply:
hope I covered it now